opencti-platform/opencti-graphql/src/domain/stixDomainObject.js

```diff
@@ -131,6 +131,13 @@ export const stixDomainObjectExportAsk = async (context, user, stixDomainObjectI
   const works = await askEntityExport(context, user, format, entity, exportType, maxMarkingDefinition);
   return works.map((w) => workToExportFile(w));
 };
+
+export const stixDomainObjectMimeTypeFilter = (stixDomainObject, mimeType = '') => {
+  // retrieve the mime type of a file in extension x_opencti_files and filter only the mimeType of your choice
+  const getMimeType = stixDomainObject.x_opencti_files;
+  const filterMimeType = getMimeType?.filter((n) => n.mime_type.includes(mimeType));
+  return filterMimeType ?? [];
+};
```

Comment:
I don't understand this method, from what I read, it returns a list of `stixDomainObject.x_opencti_files` filtered by mimeType, but the comment, name of the method and variable names are not clear.

Maybe it could be named `stixDomainObjectFiles` ?

Reply:
```suggestion
export const stixDomainObjectFiles = (stixDomainObject, mimeTypeFilter = '') => {
  const files = stixDomainObject.x_opencti_files ?? [];
  return files.filter((n) => n.mime_type.includes(mimeTypeFilter));
};
```